core/pom.xml

```diff
@@ -470,6 +477,11 @@
             <artifactId>orchid</artifactId>
             <version>1.1</version>
         </dependency>
+        <dependency>
```

Comment:
For each dependency, we need the hash of the artifact in the maven-enforcer-plugin configuration.